lib/meandro/rules/unused_record_fields.ex

```diff
@@ -0,0 +1,150 @@
+defmodule Meandro.Rule.UnusedRecordFields do
+  @moduledoc """
+  Finds instances where a record has fields defined that are not being used.
+
+  This rule assumes that your code will never use the underlying tuple
+  structure of your records directly.
+  """
+
+  alias Meandro.Util
+
+  @behaviour Meandro.Rule
+
+  @impl Meandro.Rule
+  def analyze(files_and_asts, _options) do
+    for {file, ast} <- files_and_asts,
+        result <- analyze_file(file, ast) do
+      result
+    end
+  end
+
+  @impl Meandro.Rule
+  def is_ignored?(_pattern, _ignore_spec) do
+    false
+  end
+
+  defp analyze_file(file, ast) do
```

Comment:
Should it be named analyze_module?

Reply:
probably a rename we need to do in all the rules now that the `{module, file, ast}` split happened.